src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_client_factory.py

```diff
@@ -20,7 +20,7 @@ def get_storage_service_client(cli_ctx):
 
 def get_acr_service_client(cli_ctx, api_version=None):
     """Returns the client for managing container registries. """
-    from azure.mgmt.containerregistry import ContainerRegistryManagementClient
+    from .sdk.container_registry_management_client import ContainerRegistryManagementClient
```

Comment:
Replace the local sdk with published sdk.

Reply:
done :)